python/feldera/sql_context.py

```diff
@@ -366,6 +380,86 @@ def foreach_chunk(self, view_name: str, callback: Callable[[pandas.DataFrame, in
         handler = CallbackRunner(self.client, self.pipeline_name, view_name, callback, queue)
         handler.start()
 
+    def connect_source_kafka(
+        self,
+        table_name: str,
+        connector_name: str,
+        config: dict,
+        fmt: JSONFormat | CSVFormat
+    ):
+        """
+        Tells Feldera to read the data from the specified kafka topic.
```

Comment:
It associates a specific table with a kafka topic on a specific kafka server. The table is populated with changes from that topic.